src/index.html

```diff
@@ -293,6 +293,29 @@ <h1 class="text-2xl md:text-4xl lg:text-5xl uppercase">srcery</h1>
             </svg>
           </a>
         </section>
+        <section class="max-w-2xl mt-8 md:mt-12">
+          <h2
+            class="font-iosevka text-2xl inline-block mb-6 border-b-2 border-white"
+          >
+            Merch
+          </h2>
+          <p class="mb-4 text-justify">
+            Get our brand new merchandise featuring the Srcery color scheme
+            color scheme! We're not making any profit from the sales, so you
+            can show your support at cost. Thank you for helping us develop and
+            improve our color scheme.
```

Comment:
```suggestion
            Get our brand new merchandise featuring the Srcery color scheme!
            We're not making any profit from the sales, so you
            can show your support by wearing/using the merch. Thank you for helping us develop and
            improve our color scheme.
```

Something along the line?  Not sure if "color scheme" was ment to be written twice?

Either way, for now it will work.  @roosta is soon out with the new site.